File: LLM-Translations/GPT-4/avatar/Python/C/codeforces_231_A.c
#include <stdio.h>

int main() {
    int n, c = 0;
    scanf("%d", &n);
    for(int i = 0; i < n; i++) {
        int sum = 0;
        for(int j = 0; j < 2; j++) {
            int temp;
            scanf("%d", &temp);
            sum += temp;
        }
        if(sum > 1) {
            c++;
        }
    }
    printf("%d", c);
    return 0;
}
// End of Code

File: LLM-Translations/GPT-4/avatar/Python/Go/codeforces_231_A.go
package main

import (
	"fmt"
)

func main() {
	var n, c int
	fmt.Scan(&n)
	for i := 0; i < n; i++ {
		var l1 [3]int
		sum := 0
		for j := 0; j < 3; j++ {
			fmt.Scan(&l1[j])
			sum += l1[j]
		}
		if sum > 1 {
			c++
		}
	}
	fmt.Println(c)
}

// End of Code
